frontend/public/components/routes/create-route.tsx

```diff
@@ -342,7 +346,7 @@ class CreateRouteWithTranslation extends React.Component<
             className="pf-v5-c-form-control"
             type="text"
             onChange={this.handleChange}
-            value={this.state.path}
+            value={this.state.path ?? ''} // this.state.path can be set to undefined in componentDidMount
```

Comment:
Fixes a console error related to controlled component changing to an uncontrolled one.